src/actions/account.js

```diff
@@ -164,6 +164,29 @@ async function getAccounts(tokens, options) {
   }, Promise.resolve({}));
 }
 
+export const updatedLoggedinAddresses = tokens => async (dispatch, getState) => {
```

Comment:
I think a more descriptive name would be
```suggestion
export const updateEnabledTokensAccounts = tokens => async (dispatch, getState) => {
```